.github/workflows/build-container.yaml

```
@@ -0,0 +1,130 @@
+name: Build Container
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)